/// Files: drivers/vhost/vdpa.c
/// Fix: 870aaff92e959e29d40f9cfdb5ed06ba2fc2dae0
/// Detect-To: d6d8bb92fdde6390037bf9da174ed3ab551c04d7

virtual detect

@err@
identifier size, vdpa;
position p;
@@

vhost_vdpa_config_validate(...)
{
	...
*	long@p size = vdpa->config->get_config_size(vdpa);
	...
}

@script:python depends on detect@
p << err.p;
@@

coccilib.report.print_report(p[0], 'ERROR: CVE-2022-0998')
